submissions/t-jay-777/Hooli-style Popup/index.html

```diff
@@ -2,119 +2,122 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8">
-    <title>Hooli-style Popup</title>
-    
-    <link rel="stylesheet" href="style.css">
-</head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Hooli-style Popup</title>
+	<link rel="stylesheet" href="style.css"> </head>
 
 <body>
-       <div class="container">
-        <div class="nav">
-            <ul class="nav-menu">
-                <li class="nav-submenu">
-                    <a href="#">Mail</a>
-                </li>
-                <li class="nav-submenu">
-                    <a href="#">Images</a>
-                </li>
-                <li class="nav-submenu popup">
-                    <a href="#popup-menu"><img src="img/apps.jpg" alt="apps"></a>
-                    <div class="popup-menu-1">
+	<div class="container">
+		<nav class="nav">
+			<ul class="nav-menu">
+				<li class="nav-submenu"> <a href="#">Mail</a>
+					<br>
+					<br>
+				</li>
+				<li class="nav-submenu"> <a href="#">Images</a>
+					<br>
+					<br>
+				</li>
+				<li class="nav-submenu popup">
+					<a href="#popup-menu"><img src="img/apps.jpg" alt="apps"></a>
+					<div class="popup-menu-1">
 						<div class="popup-menu" id="popup-menu">
-
 							<ul class="nonscroll">
 								<li>
-									<a href="#"><img src="img/calendar.png" alt="calendar"></a>
-									<span>Calendar</span>
-								</li>
+									<a href="#"><img src="img/calendar.png" alt="calendar"></a> <span>Calendar</span>
+									<br>
```

Comment:
Avoid using `<br>` for spacing use margin or padding for that.

Reply:
I tried just to make empty lines👀